ctapipe/tools/tests/test_tools.py

```diff
@@ -121,22 +124,32 @@ def test_merge(tmpdir):
         run_tool(
             MergeTool(),
             argv=[
-                f"{input_file_1}",
-                f"{input_file_2}",
-                f"--o={out_all}",
+                f"--i={tmp_dir.name}",
+                "--p='test_file_*.hdf5'",
+                f"--o={out_tels_dir_pattern}",
                 "--overwrite",
+                "--t=[2, 3]",
             ],
             cwd=tmpdir,
         )
         == 0
     )
 
+    assert (
+        run_tool(
+            MergeTool(),
+            argv=[f"{in_1}", f"{in_2}", f"--o={out_all}", "--overwrite"],
+            cwd=tmpdir,
+        )
+        == 0
+    )
+
     assert (
         run_tool(
             MergeTool(),
             argv=[
-                f"{input_file_1}",
-                f"{input_file_2}",
+                f"{in_1}",
```

Comment:
unnecessary fstring, since `in_1` is already a string.

Reply:
"He added an unneccesary f-string (Wasn't me)"